h/browser/chrome/help/index.html

```diff
@@ -105,6 +105,13 @@ <h1 class="help-item__heading">We’re sorry, Hypothesis doesn’t work on this
         <img class="help-item__icon" src="sad-annotation.svg" />
         <h1 class="help-item__heading">We’re sorry, Hypothesis doesn't work on this site yet.</h1>
       </section>
+      <section id="other-error" class="help-item">
+        <img class="help-item__icon" src="sad-annotation.svg" />
+        <p class="center">
+          There was a problem loading Hypothesis into this page:
```

Comment:
By all means make it shorter! But it does need to give them guidance on what they can do next, not just "There was an error: TRACEBACK" ;)